time-tracker-webapi/test/TimeTracker.Library.Test/Services/AdminReportServiceTest.cs

```diff
@@ -10,48 +10,46 @@
 
 namespace TimeTracker.Library.Test.Services
 {
-    public class AdminReportServiceTest
+    public class AdminReportServiceTest : IClassFixture<InMemoryDatabaseWithProjectsAndUsers>, IAsyncLifetime
     {
```

Comment:
What does IAsyncLifetime do here?

Reply:
IAsyncLifetime provides setup/teardown methods for xUnit that can run async tasks.